puppet/modules/freight/files/deb-HEADER.html

```diff
@@ -1,32 +1,30 @@
-<title>The Debian Foreman repo</title>
-<head><h1>The Debian Foreman repo</h1></head>
-<body>
-<p>The distros have changed!</p>
+<h1>The Debian Foreman repo</h1>
 
 <p>You will find them at:</p>
 
 <pre>deb http://deb.theforeman.org/ &lt;codename&gt; &lt;component&gt;</pre>
```

Comment:
What's a `component` in this case? I mean yeah, that's the Debian speak for main/contrib/non-free, but we use more of a `version` here, right?